microsoftgraph/microsoft-graph-explorer-v4
# Task: add English file transfer automation

## Overview

When the GE.json file is updated, a commit is made to the content repository with the latest changes

### Demo

Creates a commit with the latest change in the [microsoft-graph-devx-content/](https://github.com/microsoftgraph/microsoft-graph-devx-content/) repository

![image](https://user-images.githubusercontent.com/58787602/130664958-74a190bb-0516-4f43-8ebb-0890a1e161ab.png)

Once a change is made to a localized file, a PR like [this](https://github.com/microsoftgraph/microsoft-graph-explorer-v4/pull/1076) is generated:

![image](https://user-images.githubusercontent.com/58787602/130666316-52b7c828-9c8d-4ccb-8860-6e18199e5f55.png)

## Testing Instructions

* Make a change to the GE.json file
* Ensure the github actions run
* Go to the content repository in the expected branch
* Notice a new commit
